src/components/Shared/FormElements/AddressFormComponents/PostcodeLookup.tsx

```diff
@@ -124,7 +115,7 @@ function PostcodeLookup(props: PostcodeLookupProps) {
           }`,
           postcode: "",
         },
-        ...addresses.map((addressFromApi: address) => {
+        ...content.map((addressFromApi: Address) => {
```

Comment:
@paddyduff31 , This change has caused failing tests, reverting the change resolved the issue, do we need this change? Not sure.